client/src/ui/elements/BaseThreeTooltip.tsx

```diff
@@ -1,14 +1,17 @@
 import React from "react";
 import { DojoHtml } from "./DojoHtml";
+import clsx from "clsx";
 
 type BaseThreeTooltipProps = {
   children?: React.ReactNode;
-  position?: Position;
+  position?: Position | string;
```

Comment:
so that we can add custom position

Reply:
other than the ones already defined

Reply:
Not sure there would be much cases where you would want to use a string as a position but not add it to the `Position` enum

Reply:
i think you're right, best to keep position as enum, the goal was to be able to add variations to it, because depending on the size of the model the height/width will change. But since you can just add variations into className we don't need to modify position.
have pushed the refactor

Reply:
I see your point now, yes let's keep it like this so it's not another commit and people can add their position in the `className`